admin-api-server/src/analytics/controller/analytics.controller.ts

```diff
@@ -170,6 +170,48 @@ export class AnalyticsController {
     };
   }
 
+  /**
+   * @apiGroup Analytics
+   * @api {get} /analytics/users Request analytics user information regarding email signups
+   * @apiPermission admin
+   * @apiQuery {String[]="id","address","email","consent"} [sort] URL-decoded examples: sort: [$value,"desc"] or sort: [$value,"asc"]
+   * @apiQuery {Number[]="[number, number] e.g. [10, 25]"} [range] URL-decoded example: range: [10, 25] results in 25 records from the 10th record on
+   *
+   * @apiSuccessExample Example Success-Response:
+   *    {
+   *    "count": 2,
+   *        "data": [
+   *            {
+   *             "id": 1,
+   *             "address": "any valid user address",
+   *             "email": "maxime@muster.com",
+   *             "consent": "Yes",
```

Comment:
This column should be renamed to `marketing_consent` because otherwise it's not clear enough what this consent is about.

Also, in my opinion better to make this value a `boolean` instead of a `string`.

Reply:
I suppose you could argue either way. Especially with Typescript having enumeration types that effectively type it not as a string but as either `No` or `Yes` specifically. Only realizing this now. But we may keep it as boolean if it is not too annoying on the frontend side.

Reply:
If they complain, we change it. It is a matter of minutes so I believe it is good now.